jobs/cloud_controller_worker/templates/prom_scraper_config.yml.erb

```diff
@@ -0,0 +1,7 @@
+<% if p("cc.publish_metrics") && (p("cc.prom_scraper.disabled") != true) -%>
+port: <%= p("cc.prometheus_port") %>
+source_id: "cloud_controller_worker"
+instance_id: <%= spec.id || spec.index.to_s %>
+scheme: http
```

Comment:
Is https not possible? Other components like doppler or router use https.

Reply:
On API VMs nginx is taking care of TLS termination. On worker VMs we do not have nginx. Also I don't see a benefit in security to encrypt communication on the VM when the certificates are laying around on the VM anyways, especially not for an endpoint which just exposes metrics.

Reply:
Ok, let's go without mTLS for now.

Reply:
I will check whether we can add mTLS afterwards.